src/jit/lsra.cpp

```diff
@@ -3541,51 +3399,40 @@ void LinearScan::buildUpperVectorRestoreRefPositions(GenTree*         tree,
 //
 static int ComputeOperandDstCount(GenTree* operand)
 {
-    TreeNodeInfo& operandInfo = operand->gtLsraInfo;
-
-    if (operandInfo.isLocalDefUse)
+    if (!operand->IsLIR())
```

Comment:
Hmm, how can a node not be LIR in LSRA?

Reply:
`GT_ARGPLACE` remains in the trees, even though it doesn't appear in the linear order. I think that's the only one, so one could exclude it explicitly, but I think it's cleaner this way (and this is debug-only code so the efficiency is not a prime concern).